Description: Use system's libraries instead of the embedded ones
 .
Author: Balint Reczey <balint@balintreczey.hu>
Forwarded: not-needed

--- a/bootstrap.mk
+++ b/bootstrap.mk
@@ -1,6 +1,6 @@
 BOOTSTRAP_SUBDIRS += configure.ac
 BOOTSTRAP_SUBDIRS += lib/cpluff/configure.ac
-BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
+#BOOTSTRAP_SUBDIRS += lib/gtest/configure.ac
 
 BOOTSTRAP_TARGETS=$(basename $(BOOTSTRAP_SUBDIRS))
 all: $(BOOTSTRAP_TARGETS)
--- a/configure.ac
+++ b/configure.ac
@@ -2392,18 +2392,11 @@
 ], [0])
 
 XB_CONFIG_MODULE([lib/gtest], [
-  ./configure \
     CC="$CC" \
     CXX="$CXX" \
     CFLAGS="$CFLAGS" \
     CXXFLAGS="$CXXFLAGS" \
-    --prefix="${prefix}" --includedir="${includedir}" --libdir="${libdir}" --datadir="${datadir}" \
-    --host=$host_alias \
-    --build=$build_alias \
-    --target=$target_alias \
-    --disable-shared \
-    --enable-static \
-    --with-pthreads
+    cmake .
 ], [$SKIP_CONFIG_GTEST])
 
 AC_OUTPUT
--- a/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h
+++ b/xbmc/cores/VideoPlayer/DVDInputStreams/DllDvdNav.h
@@ -34,6 +34,8 @@
  #define HAVE_CONFIG_H
  #endif
 
+ // pull in Kodi's config h instead of dvdnav's one
+ #include "xbmc/config.h"
  #include "dvdnav/dvdnav_internal.h"
  #include "dvdnav/vm.h"
  #include "dvdnav/dvd_types.h"
--- a/tools/depends/native/TexturePacker/Makefile
+++ b/tools/depends/native/TexturePacker/Makefile
@@ -9,13 +9,13 @@
 
 ifeq ($(NATIVEPLATFORM),)
   PLATFORM = native
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 else
   PLATFORM = $(NATIVEPLATFORM)
 endif
 
 ifeq ($(NATIVE_OS), linux)
-  EXTRA_CONFIGURE = --enable-static
+#  EXTRA_CONFIGURE = --enable-static
 endif
 ifeq ($(NATIVE_OS), android)
   EXTRA_CONFIGURE = --enable-static
--- a/lib/libdvd/Makefile.in
+++ b/lib/libdvd/Makefile.in
@@ -12,7 +12,7 @@
 else
   DVDNAV_A = @abs_top_srcdir@/tools/depends/target/libdvdnav/dvdnav-install/lib/libdvdnav.a
   DVDREAD_A = @abs_top_srcdir@/tools/depends/target/libdvdread/dvdread-install/lib/libdvdread.a
-  DVDCSS_A = @abs_top_srcdir@/tools/depends/target/libdvdcss/dvdcss-install/lib/libdvdcss.a
+  DVDCSS_A =
 endif
 
 SYSDIR = @abs_top_srcdir@/system/players/VideoPlayer
@@ -36,7 +36,7 @@
 ifeq ($(ARCH),arm-osx)
   BUILD_DVDCSS = 0
 else
-  BUILD_DVDCSS = 1
+  BUILD_DVDCSS = 0
 endif
 ifeq ($(BUILD_DVDCSS),1)
   SOS += libdvdcss-$(ARCH).so
