From d265f78a136f9068dee186030c4032c12dcf553a Mon Sep 17 00:00:00 2001
From: Voyager1 <voyager@xbmc.org>
Date: Sun, 14 Feb 2016 17:17:58 +0100
Subject: [PATCH 03/10] [xbmc][dvdnav] detection of dvd name

---
 src/vm/vm.c | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/src/vm/vm.c b/src/vm/vm.c
index 35d08d5..7ff285d 100644
--- a/libdvdnav-5-0-3/src/vm/vm.c
+++ b/libdvdnav-5-0-3/src/vm/vm.c
@@ -409,6 +409,13 @@ int vm_reset(vm_t *vm, const char *dvdroot,
     if(dvd_read_name(vm->dvd_name, vm->dvd_serial, dvdroot) != 1) {
       fprintf(MSG_OUT, "libdvdnav: vm: dvd_read_name failed\n");
     }
+#ifdef _XBMC
+    if (DVDUDFVolumeInfo(vm->dvd, vm->dvd_name, sizeof(vm->dvd_name), NULL, 0))
+      if (DVDISOVolumeInfo(vm->dvd, vm->dvd_name, sizeof(vm->dvd_name), NULL, 0))
+        strcpy(vm->dvd_name, "");
+
+    fprintf(MSG_OUT, "libdvdnav: vm: DVD Title: %s\n", vm->dvd_name);
+#endif
   }
   if (vm->vmgi) {
     int i, mask;
-- 
2.1.4

