From 6b8858102577a06646635ed80ee03d93c6b380be Mon Sep 17 00:00:00 2001
From: Voyager1 <voyager@xbmc.org>
Date: Sat, 20 Feb 2016 21:01:36 +0100
Subject: [PATCH 2/3] [xbmc][libdvdread] don't do symlink resolution on kodi

---
 src/dvd_reader.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/dvd_reader.c b/src/dvd_reader.c
index 5cc06d6..3c31a1c 100644
--- a/libdvdread-5-0-3/src/dvd_reader.c
+++ b/libdvdread-5-0-3/src/dvd_reader.c
@@ -432,8 +432,9 @@ static dvd_reader_t *DVDOpenCommon( const char *ppath,
     if( !(path_copy = strdup( path ) ) )
       goto DVDOpen_error;
 
-#ifndef WIN32 /* don't have fchdir, and getcwd( NULL, ... ) is strange */
+#if !defined(WIN32) && !defined(_XBMC) /* don't have fchdir, and getcwd( NULL, ... ) is strange */
               /* Also WIN32 does not have symlinks, so we don't need this bit of code. */
+              /* XBMC also doesn't need symlink resolution */
 
     /* Resolve any symlinks and get the absolute dir name. */
     {
-- 
2.1.4

