From fe6af9cc2aab9954b0071699dc5ea1ae6ae2e244 Mon Sep 17 00:00:00 2001
From: Voyager1 <voyager@xbmc.org>
Date: Sun, 14 Feb 2016 17:19:12 +0100
Subject: [PATCH 04/10] [xbmc][dvdnav] skip cell when unrecoverable

---
 src/dvdnav.c | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/src/dvdnav.c b/src/dvdnav.c
index 39c11dd..3b61059 100644
--- a/libdvdnav-5-0-3/src/dvdnav.c
+++ b/libdvdnav-5-0-3/src/dvdnav.c
@@ -564,6 +564,10 @@ dvdnav_status_t dvdnav_get_next_cache_block(dvdnav_t *this, uint8_t **buf,
         /* Decode nav into pci and dsi. Then get next VOBU info. */
         if(!dvdnav_decode_packet(*buf, &this->dsi, &this->pci)) {
           printerr("Expected NAV packet but none found.");
+#ifdef _XBMC
+          /* skip this cell as we won't recover from this*/
+          vm_get_next_cell(this->vm);
+#endif
           pthread_mutex_unlock(&this->vm_lock);
           return DVDNAV_STATUS_ERR;
         }
@@ -860,6 +864,10 @@ dvdnav_status_t dvdnav_get_next_cache_block(dvdnav_t *this, uint8_t **buf,
     /* Decode nav into pci and dsi. Then get next VOBU info. */
     if(!dvdnav_decode_packet(*buf, &this->dsi, &this->pci)) {
       printerr("Expected NAV packet but none found.");
+#ifdef _XBMC
+      /* skip this cell as we won't recover from this*/
+      vm_get_next_cell(this->vm);
+#endif
       pthread_mutex_unlock(&this->vm_lock);
       return DVDNAV_STATUS_ERR;
     }
-- 
2.1.4

