*** frontends/framebuffer/convert_font.c	2019-07-17 11:28:13.000000000 +0100
--- frontends/framebuffer/convert_font.c.mine	2021-05-12 12:36:57.000000000 +0100
***************
*** 290,303 ****
  
  
  	for (s = 0; s < 4; s++) {
! 		fprintf(fp, "const uint8_t *%s_section_table;\n",
  			var_lables[s]);
! 		fprintf(fp, "const uint16_t *%s_sections;\n",
  			var_lables[s]);
  
  	}
  
! 	fprintf(fp, "const uint8_t *font_glyph_data;\n");
  
  	fprintf(fp, "\n\n");
  
--- 290,303 ----
  
  
  	for (s = 0; s < 4; s++) {
! 		fprintf(fp, "extern const uint8_t *%s_section_table;\n",
  			var_lables[s]);
! 		fprintf(fp, "extern const uint16_t *%s_sections;\n",
  			var_lables[s]);
  
  	}
  
! 	fprintf(fp, "extern const uint8_t *font_glyph_data;\n");
  
  	fprintf(fp, "\n\n");
  
